src/datamodel/test.rs

```diff
@@ -0,0 +1,10 @@
+use crate::datamodel::character::*;
+
+
+#[test]
+fn test_minion() {
+    let m1 = MinionOfMordred{};
+    let m2 = MinionOfMordred{};
+    assert_eq!(true, m1.is_seen_by(&m2.get_name()));
+    assert_eq!(true, m1.can_see(&m2.get_name()));
+}  
```

Comment:
We should have tests for all characters. It is OK to write them in a following PR, but in that case I suggest we track the missing ones in one issue or with `TODO` comments in the code.

Reply:
IMHO, all the code uploaded must be tested. So, I intend to cover all this new code before merging.
I'm working in a feature to block the pipeline if the coverage is below a certain threshold. So, this won't be a issue in the future.

Reply:
I agree. For this particular stage of the project where our codebase is so small that we can't work independently on different tasks, I think it is OK to sometimes merge code without tests if someone else is going to work on the tests, but in that case we should track the code that's not being tested (good to have that feature automated by your coverage analyser). For the future I think your proposal works fine.

Reply:
Thank you for your insights wachito. But, please resolve the conversation so we can go on with the remain issues. If you open it you must close it. Let's call that the rule with reviews.